File: objects/stats.go
package objects

type StatsFields struct {
	Movies struct {
		Tracked     int
		Downloading int
		Downloaded  int
		Removed     int
	}
	Shows struct {
		Tracked int
		Removed int
	}
	Episodes struct {
		Downloading int
		Downloaded  int
	}
	Notifications struct {
		Read   int
		Unread int
	}
	Runtime struct {
		GoRoutines int
		GoMaxProcs int
		NumCPU     int
	}
}

File: server/ui/src/types/stats.ts
type Stats = {
	Movies: {
		Tracked     :number,
		Downloading :number,
		Downloaded  :number,
		Removed     :number,
	}
	Shows: {
		Tracked :number,
		Removed :number,
	}
	Episodes: {
		Downloading :number,
		Downloaded  :number,
	}
	Notifications: {
		Read   :number,
		Unread :number,
	}
	Runtime: {
		GoRoutines :number,
		GoMaxProcs :number,
		NumCPU     :number,
	}
};

export default Stats;
